react/index.js

```diff
@@ -302,12 +309,14 @@ class MiniCart extends Component {
       hideContent,
       showShippingCost,
       showTotalItemsQty,
+      showPrice,
       linkState: { minicartItems: items, orderForm, isOpen },
     } = this.props
 
     const itemsToShow = this.getFilteredItems()
     const totalItemsSum = arr => arr.reduce((sum, product) => sum + product.quantity, 0)
     const quantity = showTotalItemsQty ? totalItemsSum(itemsToShow) : itemsToShow.length
+    const totalPrice = this.sumItemsPrice(itemsToShow)
```

Comment:
I think we should just get the Total value from orderForm instead of computing this "by hand"